common/domain/src/main/kotlin/no/nav/mulighetsrommet/domain/dto/TiltaksgjennomforingEksternV1Dto.kt

```diff
@@ -23,6 +23,7 @@ data class TiltaksgjennomforingEksternV1Dto(
     val oppstart: TiltaksgjennomforingOppstartstype,
     @Serializable(with = LocalDateSerializer::class)
     val tilgjengeligForArrangorFraOgMedDato: LocalDate?,
+    val apentForInnsok: Boolean,
```

Comment:
Jeg synes kanskje vi burde undersøke "åpent for innsøk" på digital jobbklubb før vi ruller ut dette.

Komet forventet ikke at det skulle være der, men vi har åpnet opp for det. 

I tillegg lurer jeg på om det er rart at vi har dette flagget for alle tiltak? Kanskje vi skulle ryddet opp i vår data og gjort dette flagget nullable, men bare påkrevd for kurstiltakene?